Tiber upgrade (#259)
autonity/docs.autonity.org: glossary/index.md

## account state
A part of [system state](/glossary/#system-state) that is specific to an [account](/glossary/#account) [address](/glossary/#address). Account state comprises: the account’s native coin balance (_auton_), the count of transactions sent from the account (nonce), and, if a contract account, a hash (storageRoot) of the associated account storage and hash (codeHash) of the associated EVM code.

## acn
Acronym for Autonity Consensus Network.

At the P2P level, Autonity separates transaction and consensus traffic on to separate channels on different TCP ports.

> For transaction gossiping between nodes the Ethereum wire protocol is used for P2P propagation of block announcements and transactions. For consensus gossiping a separate consensus protocol runs alongside the ethwire protocol for the execution of Autonity’s BFT Tendermint Consensus algorithm. For detail see [Communication layer](/concepts/architecture/#communication-layer) in the *Architecture* Concept.

Reviewer: ```suggestion
The Ethereum wire protocol is used for P2P propagation of blocks and transactions between nodes. For consensus gossiping a separate protocol runs alongside the ethwire protocol, allowing for the execution of Autonity’s BFT Tendermint Consensus algorithm. For detail see [Communication layer](/concepts/architecture/#communication-layer) in the *Architecture* Concept.
```

## address{#address}
A 64 character hex string providing the unique identifier of an [account](/glossary/#account).

## autobond
The process by which [Newton inflation emissions](/concepts/protocol-assets/newton/#total-supply-and-newton-inflation) are automatically [bonded](/glossary/#bond).

The [Newton inflation mechanism](/concepts/protocol-assets/newton/#total-supply-and-newton-inflation) emits [Newton](/glossary/#newton) [stake token](/glossary/#stake-token) as an inflationary reward for [delegated](/glossary/#delegated) [validator](/glossary/#validator) stake.

The process takes the emission earned by a [stakeholder](/glossary/#stakeholder) for stake [delegated](/glossary/#delegated) to a [validator](/glossary/#validator) and automatically [bonds](/glossary/#bond) the emission to that [validator](/glossary/#validator) in the name of that stakeholder.

The autobond process increases the Newton amount of the delegated stake of the validator without issuing new Liquid Newton tokens, resulting in an increase in the LNTN/NTN conversion ratio of the validator.

> See [delegated](/glossary/#delegated) and [self-bonded](/glossary/#self-bonded).

Reviewer: autobonding happens also for ntn rewards accrued by self bonded stake (not only the delegated one).

Author: Clarified in commit 6121df5f349d642623659b2b2952b5cdc32c51e1

## gigaton
The denomination of Autonity's [auton](/glossary/#auton) cryptocurrency used to denominate Autonity gas prices. 1 `gigaton` = 1,000,000,000 [ton](/glossary/#ton). The Autonity equivalent of wei is [ton](/glossary/#ton) and of gwei gigaton.

## inflation mechanism
The mechanism by which a percentage of the total [newton](/glossary/#newton) supply is held in an "Inflation Reserve" and released over time as [inflation rewards](/glossary/#inflation-rewards).

Newton total supply is capped at 100 million NTNs of which 40% is reserved for inflation awards in the short-to-medium term.

> Inflation rewards are emitted in the form of newton inflation (i.e. newly minted newton supply) for bonded stake [participating](/glossary/#participation-rate) in securing the [Autonity network](/glossary/#autonity-network). The quantity of newton emissions declines (in average) over time and asymptotically approaches zero as the Inflation Reserve is fixed.

Reviewer: ```suggestion
Inflation rewards are emitted in the form of newton inflation (i.e. newly minted newton supply) for bonded stake [participating](/glossary/#participation-rate) in securing the [Autonity network](/glossary/#autonity-network). The quantity of newton emissions declines (in average) over time and asymptotically approaches zero.
```

## jailbound
A state in which a validator is permanently barred from selection to the consensus committee. A validator may only enter a jailbound state if:

- A 100% [slashing penalty](/glossary/#slashing-penalty) is applied by [Accountability fault detection](/concepts/afd/#jail-period-calculation)

> - The validator is found inactive while on probation by [Omission fault detection](/concepts/ofd/#jail-period-calculation).

Reviewer: ```suggestion
- The validator is repeatedly found inactive while on probation by [Omission fault detection](/concepts/ofd/#jail-period-calculation), leading to a cumulative 100% slashing penalty.
```

Just to be more precise and not cause panic, as you cannot be jailbound if you are offline once while on probation

A participant is able to sync state, and broadcast and receive transactions, and potentially be a [validator](/glossary/#validator) node.

## participation rate

The percentage of "circulating" [newton](/glossary/#newton) supply that is bonded to registered [validator](/glossary/#validator) nodes on an [Autonity network](/glossary/#autonity-network).

> The participation rate is a key dimension of the network’s security, as it quantifies the slashing cost to an attacker successfully launching a $1/3$ superminority attack. A higher participation rate results in higher economic security.

Reviewer: This sentence is not clear to me. I think the participation rate influences the cost for an attacker to buy 1/3 of the bonded stake, however it does not influence slashing

Author: That should be staking cost not slashing cost.  The slashing risk is proportional to the amount staked, so it quantifies both for the attacker.

Reword to:

>The participation rate is a key dimension of the network’s security. It quantifies the staking cost for an attacker attempting to launch a $1/3$ superminority attack, and the slashing cost when detected. A higher participation rate results in higher economic security.

Perhaps should just remove ", and the slashing cost when detected" - less is more?

Author: Commit ba86caa121008b2e55420a953d031989d7ebb730

## unbonding period
The period of time for which bonded stake remains locked after processing an unbonding transaction. Unbonding period is defined as a number of blocks.

The unbonding period must be longer than the [epoch period](/glossary/#epoch-period).

Proof of Stake consensus places constraints on the minimum length of the unbonding period. It must be long enough to allow the detection and reporting of consensus faults by validators, and not short enough to  allow unbonding before a [slashing penalty](/glossary/#slashing-penalty) can be applied. As such it is a security property of the network.

After processing a stake [unbond](/glossary/#unbond) transaction  

> Newton is not redeemed to the stake [delegator](/glossary/#delegation) until the end of the [epoch](/glossary/#epoch) in which the unbonding period expires. 

Reviewer: Might be worth to substitute redeemed with minted also here

Author: Edited in commit 2c1d4e660ca3e38b3120d398a36a1eaa0e23bde6. Note I've reviewed all references to redemption in the glossary and where it remains it is I think OK as mint/burn is subsequently mentioned where appropriate.

The Unix OS system for representing a point in time as a timestamp. Time is measured as the number of seconds since the Unix Epoch began - 1st January 1970 at 00:00:00 UTC. Unix time is used for Autonity timestamps. For detail and format see [Unix time](https://en.wikipedia.org/wiki/Unix_time).

## validator
A [participant](/glossary/#participant) [node](/glossary/#node) that has registered as a validator on an [Autonity network](/glossary/#autonity-network). Validator nodes may be selected to the [consensus committee](/glossary/#consensus-committee) and participate in [consensus](/glossary/#consensus) if they have enough [bonded](/glossary/#bond) [stake](/glossary/#staking).

## vote period
An Autonity network's configured [voting period](/glossary/#voting-period) for price voting and aggregation by the oracle network.

## voting period
The period of time measured in [blocks](/glossary/#block) over which [Autonity oracles](/glossary/#autonity-oracle-server-aos) submit and vote on price data reports for [currency pair](/glossary/#currency-pair) symbols supported by the oracle protocol.

See [vote period](/glossary/#vote-period) and [voting round](/glossary/#voting-round).

> 

Reviewer: not a big deal, but I would rather merge these 2 to reduce potential confusion, as it seems to me that their meaning is quite similar

Author: Yes, agree.

Author: Merged voting period into vote period. Merged in commit 7f4d5b56c4be3f5a129922c3d30d893cd6ba18d6 vote period.

(Included check to update docs hyperlinks to 'voting period' and repoint to 'vote period'.)

After processing a stake [unbond](/glossary/#unbond) transaction  
Newton is not minted to the stake [delegator](/glossary/#delegation) until the end of the [epoch](/glossary/#epoch) in which the unbonding period expires. 

## Unix time
The Unix OS system for representing a point in time as a timestamp. Time is measured as the number of seconds since the Unix Epoch began - 1st January 1970 at 00:00:00 UTC. Unix time is used for Autonity timestamps. For detail and format see [Unix time](https://en.wikipedia.org/wiki/Unix_time).

## validator
A [participant](/glossary/#participant) [node](/glossary/#node) that has registered as a validator on an [Autonity network](/glossary/#autonity-network). Validator nodes may be selected to the [consensus committee](/glossary/#consensus-committee) and participate in [consensus](/glossary/#consensus) if they have enough [bonded](/glossary/#bond) [stake](/glossary/#staking).

## vote period
The period of time measured in [blocks](/glossary/#block) over which  price voting and aggregation by the [oracle network](/glossary/#oracle-network) takes place in a [voting round](/glossary/#voting-round).

> During the vote period [Autonity oracles](/glossary/#autonity-oracle-server-aos) collect price data from data providers and submit price data reports on-chain for [currency pair](/glossary/#currency-pair) symbols supported by the [oracle protocol](/concepts/oracle-network/#oracle-protocol). At the end of the vote period the oracle contract on-chain computes median price data for the symbols, publishing the data on-chain as 'round data'. A new voting round is then initiated.

Reviewer: median --> weighted average

Author: Fixed in commit 0655a73ec127c16d42c275f38bb54a172831f7bb
